hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapService.java

```diff
@@ -315,7 +315,7 @@ public Operation prepareReplicationOperation(PartitionReplicationEvent event, Co
     public void insertMigratedData(int partitionId, Map<String, Map<Data, MultiMapValue>> map) {
         for (Map.Entry<String, Map<Data, MultiMapValue>> entry : map.entrySet()) {
             String name = entry.getKey();
-            MultiMapContainer container = getOrCreateCollectionContainer(partitionId, name);
+            MultiMapContainer container = getOrCreateCollectionContainerWithoutAccess(partitionId, name);
```

Comment:
Not introduced in this PR but wanted to note this: having a single  `getOrCreateCollectionContainer` method seems to make programming easy because we will not be thinking about necessity of stats-update during container accesses.

Reply:
Yeah, that's a bit messy at the moment. The alternative would be to not implicitly call `access()` on `getOrCreateCollectionContainer()`, but do this manually in the wanted places (like we do with `update()`).

I just needed this to be correct for now. We should clean this up later though, once we have a good picture where we want which stats to be updated. We will also need a test for the transaction operations.